Libplanet.Tools/Program.cs

```diff
@@ -1,17 +1,30 @@
+using System;
+using System.IO;
 using System.Threading.Tasks;
 using Cocona;
 
 namespace Libplanet.Tools
 {
     [HasSubCommands(typeof(Apv), Description = "App protocol version utilities.")]
     [HasSubCommands(typeof(Key), Description = "Manage private keys.")]
+    [HasSubCommands(typeof(Mpt), Description = "Merkle Patricia Trie utilities.")]
     public class Program
     {
+        private static readonly string FileConfigurationServiceRoot = Path.Combine(
+            Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData),
+            "planetarium",
+            "planet");
```

Comment:
How about this (which is parallel with [@planetarium/cli][1] on npm)?


```suggestion
            "cli.json");
```

[1]: https://www.npmjs.com/package/@planetarium/cli